### PR TITLE
prepare release 3.0.0

### PR DESCRIPTION
**Pull request by mandar242**
##### SUMMARY
- Run `antsibull-changelog` to generate release notes.
- Update `galaxy.yml` version to `3.0.0`

##### ISSUE TYPE
- Release Pull Request

### CODE REVIEW

**Review by hakbailey — APPROVED** [review]

**Comment by alinabuzachis** [comment]
Before releasing, please also ensure that the links reported in the README and various roles/playbooks docs use stable-3 rather than main. This kind of update is usually done after branching and before releasing.

**Comment by hakbailey** [comment]
> Before releasing, please also ensure that the links reported in the README and various roles/playbooks docs use stable-3 rather than main.

Is there a reason we don't use relative links for these, which would prevent having to make changes like this each time we create a new release branch?

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -60,3 +60,15 @@ releases:
...
+      breaking_changes:
+      - Bump minimum version requirement for ansible-core to 2.15 (https://github.com/redhat-cop/cloud.aws_troubleshooting/pull/36).
+      minor_changes:
+      - Bump version of ansible-lint to minimum 24.7.0 (https://github.com/redhat-cop/cloud.aws_troubleshooting/pull/36).
+      release_summary: This release removes support for ansible-core < 2.15.
+    fragments:
+    - 3.0.0.yml
+    - 36-bump-ansible-lint-version.yml
+    - readme_template_update.yml
+    release_date: '2024-07-29'
```
Comment (GomathiselviS) [inline changelogs/changelog.yaml:74]: Date needs to be updated

**Review by GomathiselviS — APPROVED** [review]
Please update the release date, before you tag the release.

File: changelogs/changelog.yaml (hunk 1 above)
Comment (mandar242) [reply changelogs/changelog.yaml]: fixed

**Pull request merged**